resources/js/admin/users/components/CreateUser.vue

```diff
@@ -1,155 +1,167 @@
-
-  <template>
+<template>
     <div class="container">
-       <h1>Create New User</h1>
-       <div class="row">
-         <div class="col-8">
-           <div class="card card-body">
-               <div class="form-group">
-                 <label for="exampleInputEmail1">Username</label>
-                 <input type="email" class="form-control" id="exampleInputEmail1" aria-describedby="emailHelp">
-                 <small id="emailHelp" class="form-text text-muted">Username must be distinct</small>
-               </div>
-               <div class="form-group">
-                 <label for="exampleInputEmail1">First Name</label>
-                 <input type="email" class="form-control" id="exampleInputEmail1" aria-describedby="emailHelp">
-               </div>
-               <div class="form-group">
-                 <label for="exampleInputEmail1">Last Name</label>
-                 <input type="email" class="form-control" id="exampleInputEmail1" aria-describedby="emailHelp">
-               </div>
-               <div class="form-group">
-                 <label for="exampleInputEmail1">Email Address</label>
-                 <input type="email" class="form-control" id="exampleInputEmail1" aria-describedby="emailHelp">
-               </div>
-               <div class="form-group">
-                 <label for="exampleInputEmail1">Password</label>
-                 <input type="password" class="form-control" id="exampleInputEmail1" aria-describedby="emailHelp">
-               </div>
-               <div class="form-group">
-                 <label for="exampleInputEmail1">Confirm Password</label>
-                 <input type="password" class="form-control" id="exampleInputEmail1" aria-describedby="emailHelp">
-               </div>
-               <div id="app" class="form-group">
-                 <label>Groups</label>
-                  <multiselect
-                    v-model="value"
-                    :options="options"
-                    :multiple="true"
-                    track-by="title"
-                    :custom-label="customLabel"
-                    label="name"
-                    >
-                    
-                    <template slot="tag" slot-scope="props" >
+        <h1>Create New User</h1>
```

Comment:
Create user modal is a different ticket and is being done in a newer way. Remove this

Reply:
See #707

Reply:
I'm reviewing feature 707. Thanks.